rust/schema/ast.py

```diff
@@ -535,6 +535,7 @@ class RecordExprFieldList(AstNode, ):
 
 class RecordField(AstNode, ):
     attrs: list["Attr"] | child
+    expr: optional["Expr"] | child
```

Comment:
Do we know why this field was added?

Reply:
OK, I guess this is because of https://github.com/rust-lang/rust/issues/132162.

Reply:
indeed! it's still an experimental feature, so probably no need to do much on the analysis side (though we might want to be prepared)